### PR TITLE
oop bank account

### PR DESCRIPTION
**Pull request by Helmer85**

### CODE REVIEW

**Review by cdmichaelb — CHANGES_REQUESTED** [review]
Good work on OOP lab bank account and rectangle class. Person class still missing.

File: code/jeff/Python/oop_rectangle.py
Comment (AlexDeMichieli) [inline code/jeff/Python/oop_rectangle.py:34]: Missing the implementation of the Parallelepipede child class  . Lab description [here](https://github.com/PdxCodeGuild/class_chihuahua/blob/main/1%20Python/labs/OOP.md)

**Review by AlexDeMichieli — CHANGES_REQUESTED** [review]
Need changes before approval

**Review by AlexDeMichieli — APPROVED** [review]

**Pull request merged**
